### PR TITLE
Updated files to reflect current versions

### PR DESCRIPTION
**Pull request by rachelnicole**
Updated the generator to use the current versions of dependencies, and new scripts for generating input plugins.

### CODE REVIEW

**Comment by rachelnicole** [comment]
Removed v0.12 and v4 from the travis file to match node requirements of punchcard. Original master branch was failing with older versions of node as it is.

File: generators/input-plugin/templates/_package.json [hunk 1]
```diff
@@ -21,12 +22,13 @@
...
-    "eslint": "2.2.0",
-    "eslint-config-punchcard": "^0.1.2",
+    "ava": "^0.14.0",
+    "coveralls": "^2.11.9",
+    "eslint": "^2.10.0",
+    "eslint-config-punchcard": "^1.0.1",
     "nyc": "^6.0.0",
     "punchcard-content-types": "^0.1.0",
-    "tap-diff": "^0.1.1"
+    "tap-diff": "^0.1.1",
```
Comment (scottnath) [inline generators/input-plugin/templates/_package.json]: @rachelnicole please remove this comma. that trailing comma linter does not apply to `package.json` and this trailing comma will most likely cause errors.

**Comment by scottnath** [comment]
@rachelnicole looks good. just remove that comma and I'll merge

**Comment by rachelnicole** [comment]
@scottnath its gone!

**Pull request merged**